StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OnLoadErrorMessageHandler.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  OnLoadErrorMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+// Emitted when there is an error loading connect js
+class OnLoadErrorMessageHandler: OnSetterFunctionCalledMessageHandler<OnLoadErrorMessageHandler.Values> {
+    struct Values: Codable, Equatable {
+        let elementTagName: String
+        let error: ErrorValue
+        
+        struct ErrorValue: Codable, Equatable {
+            let type: String
+            let message: String
+        }
```

Comment:
This is still in API review, so we could address this as a followup after review is finished, but `ErrorValue` will be a public-facing `ConnectEmbeddedComponentError`:

https://docs.google.com/document/d/195BaU6k2J-2CAs9anNE6e4OlZO34N-e2HTYc6pacpTw/edit#bookmark=id.hij97vc3ka6e

So we could potentially define the public `ConnectEmbeddedComponentError` and have it conform to `Decodable`